packages/adapter-airtable/tests/index.test.ts

```diff
@@ -0,0 +1,114 @@
+import { runBasicTests } from "@next-auth/adapter-test"
+import { AirtableAdapter } from "../src/index"
+import Airtable, { Table } from "airtable"
+import dotenv from "dotenv"
+dotenv.config()
+
+/*
+
+  To run the tests you need to:
```

Comment:
I think we'll need something like a docker setup to test this on CI 🤔

Reply:
There's [NocoDB](https://github.com/nocodb/nocodb), a self-hostable open source Airtable alternative, but I don't think its API-compatible with airtable.

While your totally right here, I think as a closed source saas product it'll be really hard to pull off for Airtable. Happy to be proven wrong here though!

Reply:
Another one I just came across while googling this, [Baserow](https://baserow.io/)

Reply:
@ndom91 I agree, so what do you think, should we just ignore AirTable tests on CI then? 🤔

Reply:
I reached out to the head of developer relations at Airtable to see if I can get some time from them to sort out how to handle testing. If they don't have a resolution that can be dockerized, I'll look into mocking the API calls. In the meantime, I'll resolve the items @ThangHuuVu pointed out above so this can be merged once testing is resolved.

Thanks everyone for supporting this!